requirements/edx/github.in

```diff
@@ -58,7 +58,6 @@
 -e git+https://github.com/openedx/olxcleaner.git@2f0d6c7f126cbd69c9724b7b57a0b2565330a297#egg=olxcleaner
 git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752f35a#egg=MongoDBProxy==0.1.0+edx.2
 -e git+https://github.com/jazkarta/edx-jsme.git@690dbf75441fa91c7c4899df0b83d77f7deb5458#egg=edx-jsme
--e git+https://github.com/technige/py2neo.git@py2neo-3.1.2#egg=py2neo==3.1.2
```

Comment:
We used a github-based requirement because py2neo deleted all 3.x releases from PyPI (without a stated reason). It's not unthinkable that they'd do such a thing again, but I'm willing to take the risk and install py2neo as a real requirement again, especially since we're moving to the latest version of their package.

Reply:
So that is why py2neo wasn't in the other requirements before?

Reply:
Yup, exactly.

Reply:
That's some upgrade-forcing harshness - no doubt!